broker/scripts/session.mtu-changed.sh

```diff
@@ -7,12 +7,14 @@ NEW_MTU="$5"
 . scripts/bridge_functions.sh
 
 # Remove interface from old bridge
-brctl delif digger${OLD_MTU} $INTERFACE
+ip link set dev $INTERFACE nomaster
 
-# Change interface MTU
-ip link set dev $INTERFACE mtu $NEW_MTU
-
-# Add interface to new bridge
+# Change interface MTU and add to new bridge
 ensure_bridge digger${NEW_MTU}
-brctl addif digger${NEW_MTU} $INTERFACE
+ip link set dev $INTERFACE master digger${NEW_MTU} mtu $NEW_MTU
+
+# Turn on bridge port isolation
+bridge link set dev $INTERFACE isolated on
 
+# Bring the tunnel interface up only after port isolation is enabled
+ip link set dev $INTERFACE up
```

Comment:
I believe that just removing the interface from the old bridge does not bring the interface down.  So, I suggest just after removing the interface from the old bridge, do an ```ip link down```

Reply:
I will need to test this. Conversely, if what you say is true we could, things should continue to work even if we don't bring _up_ the interface as we add it to the bridge for the new MTU value.